src/curated-corpus/pages/CorpusItemPage/CorpusItemPage.tsx

```diff
@@ -2,16 +2,22 @@ import React from 'react';
 import { useParams } from 'react-router-dom';
 import { useApprovedCorpusItemByExternalIdQuery } from '../../../api/generatedTypes';
 import { HandleApiResponse } from '../../../_shared/components';
-import { Box, Button, ButtonGroup, Grid, Typography } from '@material-ui/core';
-import { Alert } from '@material-ui/lab';
+import {
+  Alert,
+  Box,
+  Button,
+  ButtonGroup,
+  Grid,
+  Typography,
+} from '@mui/material';
 import {
   ApprovedItemCurationHistory,
   ApprovedItemInfo,
+  EditCorpusItemAction,
   RejectCorpusItemAction,
   ScheduleCorpusItemAction,
 } from '../../components';
 import { useToggle } from '../../../_shared/hooks';
-import { EditCorpusItemAction } from '../../components/actions/EditCorpusItemAction/EditCorpusItemAction';
```

Comment:
As I was going through the components and updating MUI from `@material-ui` to `@mui` packages, I also fixed up some imports and ordered imports within components (external libraries first)